plugins/qeta/src/components/MarkdownEditor/MarkdownEditor.tsx

```diff
@@ -1,23 +1,104 @@
+/* eslint-disable no-console */
 import React from 'react';
 import ReactMde from 'react-mde';
+
+import { Config } from '@backstage/config';
 import { MarkdownContent } from '@backstage/core-components';
 import 'react-mde/lib/styles/css/react-mde.css';
 import 'react-mde/lib/styles/css/react-mde-editor.css';
 import 'react-mde/lib/styles/css/react-mde-toolbar.css';
 import { useStyles } from '../../utils/hooks';
+import FileType from 'file-type';
+import { ErrorApi, errorApiRef, useApi } from '@backstage/core-plugin-api';
+
+const DEFAULT_IMAGE_SIZE_LIMIT = 2500000;
+const SUPPORTED_FILES_TYPES = [
```

Comment:
If possible, allow these to be set in the app configuration

Reply:
Image size is already being set the types of images I can add